### PR TITLE
Ky fix versioning

### PR DESCRIPTION
**Pull request by KianYang-Lee**
Include environment.yml to ease set up of environment for repo. I have tested using this env on numpy exercises. Please proceed with testing with your respective notebooks TQ.

### CODE REVIEW

**Comment by evanyeohboonkhai** [comment]
Is that needed? We have come across the discussion that no yml file will be created because we want the student to create the environment by themself.  Is this yml file is just for the repo maintenance purpose? How about your opinion @scot-certifai ? 
@KianYang-Lee

**Comment by scotgopal** [comment]
I gave this some thought. So if we did have the environment.yml within the repo, it can be for both repo maintenance as well as for student's use. Gives them the flexibility to directly begin either numpy, matplotlib or pandas without having to go through the environment setup video. 

Despite earlier we all conceded to not have it, guess I'm okay with us having it too. Btw, I've tested the .yml and there's no problem for Pandas. @KianYang-Lee

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
+name: basics-of-data-analytics
+channels:
+  - defaults
+dependencies:
```
Comment (scotgopal) [inline environment.yml:4]: Though I didn't have any issues while creating the environment, guess we can clean this to only have the main packages like numpy, pandas, notebook, matplotlib and their respective versions only (like in time-series-labs). 

The thing with creating an environment with a file is when there's a single error during installing, it halts the whole installation process. Having only the main packages avoids such issues.

**Review by scotgopal — APPROVED** [review]

**Comment by KianYang-Lee** [comment]
@evanyeohboonkhai Without an environment.yml, how do you suggest to keep track with versioning @evanyeohboonkhai ? If Matplotlib goes through a major version update, your code will most probably break.

File: environment.yml (hunk 1 above)
Comment (KianYang-Lee) [reply environment.yml:4]: These are jupyter notebooks related packages so I include it in

**Comment by KianYang-Lee** [comment]
is this new one better @scot-certifai ? I just removed the builds

File: environment.yml (hunk 1 above)
Comment (scotgopal) [reply environment.yml:4]: It looks much better now. Thanks @KianYang-Lee . 👍

**Review by scotgopal — APPROVED** [review]

**Pull request merged**